lib/App/index.js

```diff
@@ -311,6 +308,43 @@ class App {
 		this.debug(`Validated successfully`);
 		
 	}
+
+    _checkZwaveForSetting(driver, setting) {
+        if (setting.zwave &&
+            setting.zwave.index &&
+            setting.zwave.size &&
+            setting.attr &&
+            setting.attr.max) {
+            let signed;
+            let size = setting.zwave.size;
+            let max = setting.attr.max;
```

Comment:
`let max = setting.max || setting.attr.max`

Reply:
Got it, will fix